<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="GoodDay" failures="1" tests="1" name="Default test(failed)" time="0.103" errors="0" timestamp="2022-08-18T21:44:14 IST">
  <testcase classname="Failed_testcase.b" name="TC3" time="0.018">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Failed_testcase.b.TC3(b.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC3 -->
  <testcase classname="Failed_testcase.b" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
  <testcase classname="Failed_testcase.b" name="TC2" time="0.0">
    <ignored/>
  </testcase> <!-- TC2 -->
</testsuite> <!-- Default test(failed) -->
